src/components/[guild]/Requirements/components/Requirement.tsx

```diff
@@ -69,7 +63,9 @@ const Requirement = ({
       <VStack alignItems={"flex-start"} alignSelf="center" spacing={1.5}>
         <ChildrenWrapper {...wrapperProps} display="inline-block">
           {requirement?.isNegated && <Tag mr="2">DON'T</Tag>}
-          {requirement?.data?.customName || children}
+          {requirement?.type === "LINK_VISIT"
```

Comment:
I don't really like this, had another (more generalized) idea, but I ended up using this approach. More details in my PR comment in `VisitLinkRequirement.tsx`